gatsby-node.js

```diff
@@ -8,6 +8,44 @@
 
 var path = require("path")
 
+exports.createPages = ({ actions, graphql }) => {
+  const { createPage } = actions
+  const applyTemplate = path.resolve(`src/templates/apply.js`)
+  return graphql(`
+    {
+      allMarkdownRemark(filter: { fileAbsolutePath: { regex: "/apply/" } }) {
+        edges {
+          node {
+            frontmatter {
+              role
+            }
+          }
+        }
+      }
+    }
+  `).then(result => {
+    if (result.errors) {
```

Comment:
Wait, so if we mess up the graphql, this will just break the build right? Not the site itself?

Reply:
Yeah this is all build time